### DISCUSSION TITLE
Reset symbols

### DISCUSSION THREAD

**Discussion by Artim96** [rspamd/rspamd#4654] [General]
Hey, is there a way to reset the values of all symbols to have rspamd learn from scratch? Maybe I've changed the values of too many symbols manually so some spam mails are just not recognized. Removing rspamd with apt purge, then reinstalling it and only copying back the contents of local.d and override.d (where I never set any smbols) didn't do the trick. So where are the symbols values stored? Purging Redis only deleted the bayesian statistics.

**Comment by moisseev** [comment]
You need to delete `/var/db/rspamd/rspamd_dynamic` or `/var/lib/rspamd/rspamd_dynamic` depending on file system hierarchy.

See also [What data should be backed up?](https://rspamd.com/doc/faq.html#what-data-should-be-backed-up)

**Comment by Artim96** [comment]
I don't think this helped. I even went as far as deleting all rspamd-related directories present in its .deb package (I'm on Debian 12 if that helps, but I'm getting rspamd from their repository, not from Debian). The statistics about how many emails have been scanned are gone now, but some symbols still seem to have values that I gave them. I even reinstalled rspamd once again and flushed the whole redis database, no change.

**Comment by moisseev** [comment]
If the `rspamc counters` command shows default symbol scores but the WebUI Symbols tab shows different scores then definitely they are stored in the dynamic configuration.

If you cannot find it in `/var/lib/rspamd/` probably you should ask package maintainers where the `DBDIR` is located.
